electron/debugging/docker.md

```diff
@@ -0,0 +1,52 @@
+## Debugging CI With Docker
+
+Sometimes, you may experience test discrepancies between your local machine and the ones used to run CI, and so want to replicate those failures in an environment that most closely resembles them.
+
+On Linux, that's going to require you to run tests in a Docker container.
+
+1. Ensure you have Docker installed on your Linux machine.
+    * `$ sudo apt install docker.io`
+2. Ensure the Docker daemon is running 
+    * `$ sudo systemctl start docker`
+3. Find Electron's Docker image in our CircleCI configuration - it should look something like:
+    * `electronjs/build:3bba0fdee0d5650e751c2561f1806f9138dfe56a`
+3. Mount the Docker container:
+
+```sh
+$ docker run -it --privileged \
+    --mount type=bind,source=/path/to/electron-gn,target=/tmp/electron-gn \
+    --rm electronjs/build:3bba0fdee0d5650e751c2561f1806f9138dfe56a
+```
+
+Where `source=/path/to/electron-gn` should be replaced with the path to the directory on your computer that contains `src`, and `target=/tmp/electron-gn` is the point at which that directory will be mounted in the Docker container.
+
+After this command has finished executing, you should find yourself in the Docker container on the command line:
+
+```sh
+builduser@193defac9168:$
+```
+
+We chose to mount the Electron directory to `/tmp/electron-gn`, so we should now run:
```

Comment:
Should add for clarity when steps are to be run inside of Docker, or when they should be run in a native shell